### DISCUSSION TITLE
Possible values of Status for GET backup/status api

### DISCUSSION THREAD

**Discussion by VaibAg** [Altinity/clickhouse-backup#506] [Q&A] [answered]
Get Backup Status API returns a response of the form
// ActionStatus Struct for Backup Status
type ActionStatus struct {
	Command string `json:"command"`
	Status  string `json:"status"`
	Start   string `json:"start,omitempty"`
	Finish  string `json:"finish,omitempty"`
	Error   string `json:"error,omitempty"`
}

I wanted to know what are the possible values for Status apart from "in progress" and "success"

Thank you

**Comment by Slach** [comment, marked as answer]
currently, `success`, `in progess`, 'error` but could be add new values in future

**Reply by VaibAg** [reply]
Thank you!
